onnx/backend/test/case/node/mul.py

```diff
@@ -34,6 +34,12 @@ def export():  # type: () -> None
         expect(node, inputs=[x, y], outputs=[z],
                name='test_mul')
 
+        x = np.random.randint(24, size=(3, 4, 5), dtype=np.uint8)
+        y = np.random.randint(24, size=(3, 4, 5), dtype=np.uint8)
+        z = x * y
```

Comment:
not sure if this can be treated as a valid test case
1. max allowed value here is 24 and 24 * 24 will overflow. In this case different hardware can produce different results and it may not match the expected result in output
2. I think because we don't handle overflow we should not add a test case with expected result when there is an overflow

Reply:
good point, updating the test case.